### PR TITLE
removes to clipboard from translations

### PR DESCRIPTION
**Pull request by decanus**
non messed up version of: https://github.com/status-im/status-react/pull/7704

### CODE REVIEW

**Comment by mandrigin** [comment]
@decanus this one still has 4 commits :(

**Comment by decanus** [comment]
@mandrigin seems to be done but it now makes the merge commit weird? Enabling squash merging on the repo would make this easier for less talented git folk.

**Comment by mandrigin** [comment]
@decanus yeah, I know 😞

**Comment by mandrigin** [comment]
@decanus okay, that is what removed the merge commit for me on your branch.

just do `git rebase develop` and then `git push -f origin HEAD` from your PR branch and the merge commit should be gone.

**Comment by decanus** [comment]
awesome thanks @mandrigin
image

I'd suggest enabling only `Allow squash merging` or `Allow rebase merging `, then it doesn't require all the manual work.

**Comment by mandrigin** [comment]
@decanus it's a bit different story there, we merge it using a special script that all the commits are GPG signed in this repo. GitHub doesn't allow to do that with this button as far as I know...

**Comment by mandrigin** [comment]
@annadanchenko @Serhy it would be cool if we re-run e2e on this PR, no need to thoroughly test, because it is a version of https://github.com/status-im/status-react/pull/7704 that was corrupted.

**Comment by decanus** [comment]
@mandrigin when you press the squash merge button github uses its gpg key. See: https://github.com/yeeth/BeaconChain.swift/commits/master

**Review by mandrigin — APPROVED** [review]

**Comment by mandrigin** [comment]
@decanus oh, thanks, didn't know that, then we should definitely switch to that.

**Comment by jeluard** [comment]
@hesterbruikman Any opinion on that one?

**Comment by jeluard** [comment]
My bad, just saw the original issue

**Review by jeluard — APPROVED** [review]

**Comment by jeluard** [comment]
@decanus Not when it's a remote contributor/repo, at least that was the case before.

File: translations/en.json [hunk 1]
```diff
@@ -978,7 +978,5 @@
     "mobile-network-go-to-settings": "Go to settings",
     "mobile-network-use-mobile-data": "Status tends to use a lot of data when syncing chats. You can choose not to sync when on mobile network",
     "status-not-sent": "",
-    "status-not-sent-without-tap": "",
-    "confirm-install": "Confirm Install",
-    "extension-install-alert": "Development mode is required to install an extension. Do you want to enable and continue installing?"
+    "status-not-sent-without-tap": ""
```
Comment (flexsurfer) [inline translations/en.json]: looks like this was removed during rebasing

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: translations/en.json (hunk 1 above)
Comment (decanus) [reply translations/en.json]: It doesn't seem to exist in the develop branch anymore.

**Comment by churik** [comment]
@decanus don't see in this commit all required changes from https://github.com/status-im/status-react/pull/7704#issuecomment-472848372 - I mean test changes.
Thanks!

File: translations/en.json (hunk 1 above)
Comment (flexsurfer) [reply translations/en.json]: i mean last two items "confirm-install" and "extension-install-alert" we have them in develop https://github.com/status-im/status-react/blob/8e1da91ea01b7eb0d7fd5203df4485a12f94f7bf/translations/en.json#L982

**Comment by decanus** [comment]
@churik, done

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
one test is failed due to https://github.com/status-im/status-react/issues/7750, so PR is ready to merge

**Pull request merged**